Source/DiscoveryV1/Models/Gateway.swift

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright IBM Corporation 2018
```

Comment:
Something i need to start checking - this should be 2019. I think Mike was considering a strategy for this at some point.